src/nIsForName.js

```diff
@@ -22,7 +22,11 @@
  * @return {string[]}
  */
 function nIsForName(names) {
-  // write code here
+  return names.filter(
+    fullName => {
+      return fullName.split(' ').some(name => name[0] === 'N');
```

Comment:
use short version without return for `.filter()`

Reply:
nailed it!

Reply:
Thanks!